### PR TITLE
Changed the project organization

### PR DESCRIPTION
**Pull request by niedbalski**
- Created a setup.py file for the project
- Created test-requirements.txt and requirements.txt for tracking project dependencies
- Moved files (across all the project)
- Added gunicorn.conf.py for running the project with gunicorn (ftw).
- Added a proper tox.ini

Pending:
- Readme File

(env)niedbalski@theos-mobile:~/src/pet/os-reststack-manager$ tox 
flake8 installed: 
flake8 runtests: PYTHONHASHSEED='4147456246'
__________________________________________________________________________________________________________________________ summary ___________________________________________________________________________________________________________________________
  flake8: commands succeeded
  congratulations :)

### CODE REVIEW

File: os_reststack_manager/app/__init__.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+app.config.from_object('os_reststack_manager.config')
+
+# DB setup
+db = SQLAlchemy(app)
+db.create_all()
+
+credentials = parse_config(CONF.CREDENTIALS)
+
+
+class Tenant(db.Model):
```
Comment (gemagomez) [inline os_reststack_manager/app/__init__.py:21]: This class needs to be between lines 16 and 17 or else the db won't have the right schema on creation.

File: os_reststack_manager/logs/tenant_manager.log [hunk 2]
```diff
@@ -0,0 +1,15 @@
+02-02 15:04:56 os_reststack_manager.app ERROR    Exception on /tenant/1 [GET]
```
Comment (gemagomez) [inline os_reststack_manager/logs/tenant_manager.log]: This is a log file, shouldn't be in the branch. This is probably an issue with .gitignore.

File: run.py [hunk 3]
```diff
@@ -1,6 +0,0 @@
-#!/usr/bin/env python
```
Comment (gemagomez) [inline run.py:1]: Is there a reason not to have the option to run without gunicorn? I'd like to keep this file.

File: os_reststack_manager/config/cloud_config.yml.sample [hunk 4]
```diff
@@ -0,0 +1,4 @@
+extra_files:
```
Comment (gemagomez) [inline os_reststack_manager/config/cloud_config.yml.sample]: The three yaml files in the config directory are now duplicated, they have a .yml.sample and a .yml version. I don't think this is wise, the service should fail if it hasn't been configured by the admin, that is why the .yml file were missing, to force the person installing to configure the app. I think the .yml files should be removed.

File: run.py (hunk 3 above)
Comment (niedbalski) [inline run.py:1]: I added a console_script, this can be invoked after installing the package with 'os-reststack-manager'

**Comment by gemagomez** [comment]
Please, move the config and the templates directory one level up, they shouldn't be part of the package they are documentation, rather. It is up to the user to define those for their particular use case.

**Comment by niedbalski** [comment]
Done

**Pull request merged**